library/alloc/tests/lib.rs

```diff
@@ -1,5 +1,6 @@
 #![feature(allocator_api)]
 #![feature(box_syntax)]
+#![feature(cow_is_borrowed)]
```

Comment:
Should this have `cfg(test)`? Not sure what the style is.

Reply:
As far as I understand it everything under `library/alloc/tests` is already only included for tests. 
https://github.com/rust-lang/rust/blob/538e198193451e680cc95b66c82baac3ac687c8c/library/alloc/Cargo.toml#L17-L19